content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,317 @@
+  </tr>
+  <tr>
+    <th>Pictograms</th>
+    <td><img src="https://user-images.githubusercontent.com/6744014/89239387-dce0cd00-d5ad-11ea-898b-b574bb837836.png"/>
+    <img src="https://user-images.githubusercontent.com/6744014/89239389-dd796380-d5ad-11ea-84f4-c0819100fc2a.png"/></td>
+    <td>Use pictograms to represent the feature.</td>
+  </tr>
+  <tr>
+    <th>Illustrations and gifs</th>
+    <td><img src="https://user-images.githubusercontent.com/6744014/90083962-d39be280-dcc8-11ea-9d38-4bc24f759ba9.gif"/></td>
+    <td>Visually communicate or demonstrate a complex feature with illustrations and gifs to supplement a message.</td>
+  </tr>
+</table>
+
+## Inline feature discovery
+
+Inline feature discovery is a way to highlight features without obscuring other parts of the page while a user may be performing an existing task. 
+
+### Page banner
+
+Use page banners for annoucements that are relevant to the core task on the current page. Remember to consider using other feature announcement patterns or marketing strategies instead of disturbing the user with multiple announcements.
+
+Include a clear way for users to dismiss a page banner and [clear campaign parameters](#set-clear-campaign-limits).
+
+<Flex alignItems="center" flexDirection="column" my={3}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="green.5" mb="2" overflow="hidden">
+    <img src="https://user-images.githubusercontent.com/6744014/88745191-40768080-d0fe-11ea-97ae-6d9139027646.png" />
+  </BorderBox>
+  <p><Text fontWeight="bold" color="green.5">Do:</Text> Call out a major feature or change in a page with an invitation to engage and a clear way to dismiss.</p>
+</Flex>
+
+<Flex alignItems="center" flexDirection="column" my={3}>
+  <BorderBox width={12 / 12} border="1px solid" borderColor="red.5" mb="2" overflow="hidden">
+    <img src="https://user-images.githubusercontent.com/6744014/88744095-3acb6b80-d0fb-11ea-876e-c461ee96de7c.png" />
+  </BorderBox>
+  <p><Text fontWeight="bold" color="red.5">Don't:</Text> Don't use a page banner to disproportionately highlight a small feature.</p>
+</Flex>
+
+### Inline banner
+
+Use inline banners in a page with multiple steps or actions.
+
+<table>
+  <tr>
+    <td></td>
+    <th>Example</th>
+    <th>Notes</th>
+  </tr>
+  <tr>
+    <th>Inline feature banner</th>
+    <td>
+      <img src="https://user-images.githubusercontent.com/6744014/88744364-f5f40480-d0fb-11ea-97c4-5ab87b548a2c.png"/>
+      <p>A short introduction to the Wiki feature while configuring repository settings.</p>
+    </td>
+    <td>Organization, repository, or user settings, where a feature is managed and customized, inline alert banners can highlight relevant features that may help better accomplish a user’s task.</td>
+  </tr>
+  <tr>
+    <th>Hidden revealed</th>
+    <td>
+      <img src="https://user-images.githubusercontent.com/6744014/89241458-d35a6380-d5b3-11ea-9307-52c8ae80f2d3.png"/>
```

Comment:
@colebemis Do you have an idea of how we can make these images bigger if the reader wants to see a bigger version? Is there a simple way to do that in Doctocat?